Assets/LeapMotionModules/UI/Scripts/Constraints/Abstract/ConstraintBase.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Leap.Unity;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.UI.Constraints {
+
+  public enum Axis   { X, Y, Z }
+  public enum Anchor { Beginning, Center, End }
+
+  public static class ConstraintExtensions {
```

Comment:
This looks like it belongs in a generic util class somewhere, not in constraints.